src/CallbackController.php

```diff
@@ -31,11 +35,30 @@ public function callback(Request $request)
         if (!$request->has('code')) {
             throw new AuthenticationException('No authorization code received');
         }
-        $token = $this->provider->getAccessToken('authorization_code', [
+        $token = $this->provider->getAccessToken(
+            'authorization_code',
+            [
             'code' => $request->get('code'),
-        ]);
+            ]
+        );
```

Comment:
unrelated change, `[`, `]` and it's key on the same column looks bad, i prefer the original one